### DISCUSSION TITLE
posts/2023/06/shenzhen/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#7] [General]
# posts/2023/06/shenzhen/

https://guozheng.netlify.app/posts/2023/06/shenzhen/

**Comment by CyrusYip** [comment]
> 这种陌生人之间的搭话和求助，已经不知不觉成为一种风险的标志。

想起了扶老人被敲诈的事件还有近年层出不穷的诈骗，这些看多了就很难假定陌生人是好人。

> 她和我说没有散钱了，微信里也没有，无法坐地铁和打车回去，想找我帮忙，给她大概十七八块钱坐车。

也许可以让她开通微信的分付（微信自带的借贷服务），这样她就有钱用了。

**Reply by residualsun1** [reply]
> 也许可以让她开通微信的分付（微信自带的借贷服务），这样她就有钱用了。

我去查了一下，微信分付还没有全部开放，系统要根据用户信用分等情况进行评估才会进一步开通。不过这不失为一种帮助陌生人的思路，学到啦😄。

我这段时间都是凌晨更新博客，其实帖子大部分都没有写完，但因为害怕把当天的事情忘记，所以都会起一个头。这次的地铁借钱事件，其实更多让我反复去想，我应该从什么角度思考自己和他人的信任关系，我觉得不能只是和过去一样“一笔带过”，只是一刻的犹豫，然后选择收回信任，然后就没有下文了。我们或许可以探索一些勇敢的尝试，失败一两次也没关系（比如这次，我想即使是被骗走了十七八块也是可以接受的）。

**Reply by CyrusYip** [reply]
> 我去查了一下，微信分付还没有全部开放，系统要根据用户信用分等情况进行评估才会进一步开通。

我猜经常用微信支付的人信用还是合格的。

> 我这段时间都是凌晨更新博客，其实帖子大部分都没有写完，但因为害怕把当天的事情忘记，所以都会起一个头。

可以先写到笔记软件，我现在都是在 Joplin 写草稿，完善后再发表。很多灵感都稍纵即逝，最好先记下来。

> 我们或许可以探索一些勇敢的尝试，失败一两次也没关系（比如这次，我想即使是被骗走了十七八块也是可以接受的）。

虽然没有十七八块坐车有点奇怪，但以此骗钱那也太低效了，一天赚不了几个钱。也许她真的有困难。

**Reply by residualsun1** [reply]
> 我猜经常用微信支付的人信用还是合格的。

感觉标准挺高的，我现在支付分有590，但是还没有分付的功能。

> 可以先写到笔记软件，我现在都是在 Joplin 写草稿，完善后再发表。很多灵感都稍纵即逝，最好先记下来。

借着这个机会，我可以再学一些编程知识，十分乐意去了解和学习！

> 虽然没有十七八块坐车有点奇怪，但以此骗钱那也太低效了，一天赚不了几个钱。也许她真的有困难。

其实现在回想起来感到很懊悔，我记得最后她还说了“还是等我儿子开车来接我”（她一开始求助是因为不好意思找家里人帮忙，但我记不清更具体的原因了），应该不会有很大问题。这次就当作一个教训了。

**Comment by liechi** [comment]
看完你写的这个故事，我倒是有点担心那位老大姐的精神状况。你问的关于家人和是否能加微信这两个问题在我看来都很平常且合理，一般来说不会激起这么强烈的反感；这位老大姐似乎情绪不大稳定。我平时很少加别人微信，但设身处地想，如果我是求助的那个人，应该会主动请求加上微信，方便脱困后还钱。

想起一件多年前的事。那时我在北京，老师让去一个比较远的地方取回一台维修的仪器，一路需要坐公交、转地铁。走的时候比较匆忙，取完机器准备返程时发现身上没钱了；更要命的是，手机也没了电，无法找朋友救急。站在地铁售票口，无可奈何，最后只能硬着头皮去向售票处的一位大姐求助，希望她能借我点钱回学校。那位大姐说，她工作期间身上不能带自己的钱，当然也不能动她收的票钱，所以没法帮我。她提议说可以让我进站上车，只是到站后怎么出去得自己想办法了。正在这时候，一位穿制服的年轻姑娘过来问怎么回事。她好像是负责那个站口安检的，跟大姐相识。大姐跟她说了一下我的情况，然后她就摸出钱包，说可以为我买票。我如蒙大赦，但随即想到路上还得转一班公交车才能回到学校，于是只能再次硬着头皮问她能否再借点钱给我坐公交。她也没有犹豫就帮了我。当天到学校都比较晚了，所以第二天才辗转回到那个地铁站还钱。但找了一圈没有看到那个姑娘，售票大姐说她今天不在这里上班。我当时请大姐帮我把钱还给她，但忘了大姐是否答应。因为按大姐之前说的规定，工作时间里她似乎不能经手和票务无关的钱。虽然钱不多，但帮我解了燃眉之急；而且那种被陌生人帮助后感到的好意到现在想起来都觉得心里暖暖的。

在你另一篇博文里你提到要去浙江大学参加人类学训练营。当时想也是巧合，我对人类学了解不多，所知的两个相关工作都恰好和浙江有点关系---费孝通的《江村经济》所记录的开弦弓村毗邻浙江，项飚关于流动人口的早期研究也以北京“浙江村”为对象。后来看项飚所提的消失的“附近”这个概念有感，还写过一篇文章。本来想留个言，后来想离题太远，于是作罢；另一个没有留言的原因是你在那篇文章里说了我的好话，我觉得却之不恭，受之有愧。留言的话，又不好装作没有看到。正好你在本文末也提到了“附近”，哈哈哈，真是巧了。

**Reply by residualsun1** [reply]
> 这位老大姐似乎情绪不大稳定

对，我当时也有这种感觉，你把我没在帖子里描述的感受补充了回来。

> 而且那种被陌生人帮助后感到的好意到现在想起来都觉得心里暖暖的。

唉，看完你的这段经历，我更后悔当时没有果断一些选择帮助那位阿姨。

>另一个没有留言的原因是你在那篇文章里说了我的好话，我觉得却之不恭，受之有愧.

哈哈，可能存在一点“个人崇拜”的色彩在里面。不过可不是我一个人夸你~

> 正好你在本文末也提到了“附近”，哈哈哈，真是巧了。

现在再回想一下，我当时很明显的反应也是引用文章里讲到的**不知所措**。在不知所措后，我还是选择了拒绝信任。这种不知所措恰恰揭示了我和他人之间联结的脆弱性。我想，我们和他人之间的联结本来是维持一种正常的淡淡的界限，有时会因为一方的主动和善意而模糊了界限，清晰了情感，就像你回复中的和陌生姑娘的故事。可能也是因为我的社会阅历还非常浅薄，我觉得这样的故事在当下很少见了，如果发生也会觉得非常非常难得、非常非常珍贵，这或许又恰恰说明我们现在面临着“附近”缺失的问题。

质性研究方法课上，我们大家讨论过“附近”的话题，有同学就发问：

>既然项飙老师说要重建我们的“附近”，那“附近”是从什么时候开始消失的呢？

我当时没有答案。但我现在觉得可以尝试给出一点自己浅薄的理解：我想可以把视角放在市场经济改革带来的一系列影响。原因在于，我对阿瑟·克莱曼在《道德的重量》一书中叙述的严医生的故事印象非常深刻。这段故事大概是讲严医生如何见证从集体化时期到市场化时期，国家主导的社会观念发生了怎样剧烈的变迁。过去强调团结、集体主义，我为人人，人人为我，但后来很多东西似乎一夜之间都变了，发展紧紧围绕着金钱和商品打转。“这个时代是徐伟庆式的人的时代，我的时代已经过去去。现在是无德无品之人的时代”。可能扯远了，不过我想我需要借助我这学期阅读过的为数不多的书籍来督促自己思考。这里面有相近的地方，现在大多数人都会认为关于金钱的、效率的、某种交换的世俗人情是理所当然的，举一个典型的例子，可能我们很多人都认为要真正看好病，给主治医生塞红包是必须的流程，这好像变成了心照不宣的事情。以及文中徐伟庆做的，市场经济改革后，如何让医院上市赚钱才是最重要的，而非严医生关心的如何追赶和弥补文革时期落下的医学技术，发展基础的医学知识。“这些都是落伍的理想主义了”。想到这一个是因为，我们感知道德重量的能力越来越弱了，尊重他人、爱他人的能力也因此随之弱化了，从而导致“附近”的瓦解——从前那种“我为人人，人人为我”的时期或许真的有过？小时候会听到解放军买东西时商家不在，也会把钱放好。当然这个比较特殊，受时局等因素影响。但不可忽视的，是我确实看到市场化以后，“有名气的医生拥有太多病人，所以，新的病人只有送上恰当的礼物，才能让他们看病，这常常是在这些医生身旁放一纸包或一信封的现金”。我爸妈如果没有在医疗系统认识人，是不会带我去看病的，更愿意找偏方。我从我读警校的朋友那儿也了解到，他们警校生其实时不时都会去洗脚城……我会想，以前的警察是怎样的？以前的军人是怎样的？很多我现在觉得合理的，是不是某种东西（社会学的讨论对象往往是资本主义、现代性）让我认为是这样的……而我相信，学习社会学，便是要穿过迷雾，看到被层层遮蔽的社会现实（reality），这是曼海姆口中的“知识分子”需要做的。

我有时候也分不清，我有时候觉得有点可怕，在读完《道德的重量》之后。我内心深处所感知到的接受的道德教育告诉我何谓美与善，但现实的经验道德却一次又一次冲击它，在阅读之前，我一度认为许多东西是理所当然的，无可厚非的，因为它符合社会运转（当下暂时如此），符合我过去没看到的市场的、商品的、物欲的、资本的运转逻辑，那些我们过去颂扬的奉献好像只能在学术中出现，而无法被普遍践行，被视为“迂腐的不切实际的理想主义”。但我也知道许多人也是为了生计，而不得不去顺从，但这也是**社会的结构性问题**。

这些应该都在慢慢侵蚀我们的“附近”，伤害着我们与他人的关系，产生信任缺失、本体性安全感缺失等等问题。毕竟，我们都变得太“精明”了——我们很熟悉要去防备他人、我们很熟悉社会会存在各种各样我们“习以为常”的问题，我们甚至熟悉要用一套什么模式去应对。还是举回典型例子：碰瓷、医学治疗中的红包等等。这正是让人难过的地方——社会从来就是如此吗？即使如此，也应当如此吗……矛盾的是这又不能全怪我们，确实存在利用各种手段操弄他人信任的人，我为自己的这次拒绝选择而懊悔，可能是因为才想到十七八块确实代表阿姨有困难，但更多的还是我“天然”、“下意识”地觉得他人都不要信任，而这种感受的主要来源还是媒体以及身边他人的话语，我和现实是被分割的，我作为学习社会学的学生，反而也陷入无知的被操弄的泥潭中。我本该帮助阿姨，并且能及时反思到这些内容（虽然是我的“偏见”）

絮絮叨叨了好多，限于笔力有限，阅历的浅薄还有思考程度的不足，逻辑会比较混乱，内容也很散乱。不过我都是想回到“附近”这个说法，借着《道德的重量》试图做一个思考的材料。咱们没有给“附近”界定概念，我想我是经验地把它简单当作日常生活中人与人之间的情感联系，然后从自己的经历出发，基于我浅薄的阅读理解认为宏观的社会变迁以及制度运行形塑了人们的文化观念以及行为，进而逐渐导致了“附近”瓦解。

**Reply by liechi** [reply]
> ......我作为学习社会学的学生，反而也陷入无知的被操弄的泥潭中。我本该帮助阿姨......

信任问题是社会问题，不是个人问题，冰冻三尺非一日之寒。你虽然不能做到直接给予陌生人完全的信任，但依然试图帮忙，已经很好了。说到“被操纵”，我倒是觉得你的情绪被那位大姐无意间操纵了。正如我先前所说，你问的两个问题都是平常且合理的，并没有什么不对。你之所以觉得自己做错了，是因为那位大姐的反应让你觉得你伤害了她。那位大姐的情绪似乎不稳定，而这不是你能左右的。不必过于自责，过度反思。

> 这或许又恰恰说明我们现在面临着“附近”缺失的问题。

我觉得这跟“附近”的缺失是有区别的。如果把社会看成一个画布，“附近”是以个人或家庭为中心由浓到淡的一个颜色梯度，有个大概范围；而我们碰到的情况反映的是整个画布的底色。我觉得随着社会的发展，这个底色总会再次温暖起来的。

我之所以会对“附近”这个概念有感，是因为我的成长过程就是一个“附近”逐渐消失的过程。从农村到小城市，从小城市到大城市，越走越感觉自己现实生活的外延在缩小。小时候出门，一路走都是熟人打招呼；现在把家门一关，就孑然一身了。至于原因，我也想过，但想法还不成熟。等想好了再来补充。

**Reply by residualsun1** [reply]
> 不必过于自责，过度反思。

嗯，有时候我会想得太多太远，徒增不必要的负担。

> 至于原因，我也想过，但想法还不成熟。等想好了再来补充。

诶嘿，我也得好好看书和实践，重新整理思绪。自己写出来的东西至少要逻辑清晰，内容明确，扣准主题，这样才能更好地和别人表达自己的想法。
